tests/acceptance/test_acceptance/test_config.py

```diff
@@ -136,7 +139,8 @@ def test_config(session_obj):
     as well.
     :param session_obj: session object
     """
-    resp = session_obj.get(BASE_URL + ENDPOINT_CONFIG)
+    resp = create_and_validate_request_and_response(ENDPOINT_CONFIG, 'get', session_obj)
+    # config response validations failed due to invalid schema value
```

Comment:
Config tests are failing due to invalid schema issues

Reply:
You don't need to put this as a comment in code. Github PR comments should suffice.